packages/components/autocomplete/examples/ControlledAutocomplete.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {
+  Autocomplete,
+  Stack,
+  Paragraph,
+  Button,
+} from '@contentful/f36-components';
+
+export default function ControlledAutocomplete() {
+  const spaces = [
+    { name: 'Travel Blog', type: 'Community' },
+    { name: 'Finnance Blog', type: 'Community' },
+    { name: 'Fitness App', type: 'Medium' },
+    { name: 'News Website', type: 'Medium' },
+    { name: 'eCommerce Catalogue', type: 'Large' },
+    { name: 'Photo Gallery', type: 'Large' },
+  ];
+
+  const [selectedSpace, setSelectedSpace] = React.useState({ name: '' });
+  const [filteredItems, setFilteredItems] = React.useState(spaces);
+
+  const handleInputValueChange = (value) => {
```

Comment:
yes, the item is still selected in that case, because it only erases the input value, not the actual selection.
but we can't change this behaviour right now, as it would be a breaking change.